src/main/java/igrad/logic/parser/DeleteCommandParser.java

```diff
@@ -1,10 +1,10 @@
-package seedu.address.logic.parser;
+package igrad.logic.parser;
 
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static igrad.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
```

Comment:
this is the right way! (preserving and using the static imports for these (static) constants instead)